services/ekss.yaml

```diff
@@ -13,7 +13,11 @@ produces:
     - path: /secrets/{secret_id}/envelopes/{client_pk}
       method: GET
 storage:
-  vault: true
+  vault:
```

Comment:
Instead of splitting by consuming/producing API:
```
consumes:
  events: []
  rest_endpoints: []
produces:
  events: []
  rest_endpoints:
    - path: /secrets
      method: POST
    - path: /secrets/{secret_id}/envelopes/{client_pk}
      method: GET
```

It might be better to split by type of API:
```
api:
  rest:
    produces:
      - path: /secrets
        method: POST
      - path: /secrets/{secret_id}/envelopes/{client_pk}
        method: GET
```

Because like that you can easily see whether a service uses events, rest, or both. API types not used don't need to be specified.

Reply:
Schema and yamls were updated accordingly